event-based-simulators/main/event_based_simulators.py

```diff
@@ -137,7 +143,10 @@ def __type_fragment(self, event_definition, text = None):
         }
 
     def __log_ignore(self, event_definition):
-        print(f'{self.device_id} is down -> ignore event {event_definition["type"]}')        
+        print(f'{self.device_id} is down -> ignore event {event_definition["type"]}')
+
+    def __log_not_in_shift(self):
+        log.info(f'Device: {self.device_id} is out of shift -> ignore event')
```

Comment:
as you mentioned already: this might lead to lots of (unnecessary) log messages... maybe only log once per non-PROD-timeslot?

Reply:
Done